mypy/errorcodes.py

```diff
@@ -271,7 +271,7 @@ def __hash__(self) -> int:
 del error_codes[FILE.code]
 
 # This is a catch-all for remaining uncategorized errors.
-MISC: Final = ErrorCode("misc", "Miscellaneous other checks", "General")
+MISC: Final[ErrorCode] = ErrorCode("misc", "Miscellaneous other checks", "General")
```

Comment:
Is this because we are not deferring top-levels, LOL?

Reply:
Possibly, though I didn't look into this in any detail. This is not the first instance of this issue in this module.